tools/checked-c-convert/ProgramInfo.h

```diff
@@ -120,15 +180,16 @@ class ProgramInfo {
 
   // For each pointer type in the declaration of D, add a variable to the 
   // constraint system for that pointer type. 
-  bool addVariable(clang::Decl *D, clang::DeclStmt *St, clang::ASTContext *C);
+  bool addVariable(clang::DeclaratorDecl *D, clang::DeclStmt *St, clang::ASTContext *C);
 
   bool getDeclStmtForDecl(clang::Decl *D, clang::DeclStmt *&St);
 
   // Checks the structural type equality of two constraint variables. This is 
```

Comment:
>two constraint variables. [](start = 44, length = 25)

The arguments are now sets, now individual variables.   What does this do for the the sets of variables?